src/components/NavBar/PostButtonModal.tsx

```diff
@@ -34,6 +36,7 @@ export default function PostButtonModal({
       open={openModal}
       scroll="paper"
       sx={styles.dialog}
+      PaperProps={{ sx: { borderRadius: 5 } }}
```

Comment:
Can we look into doing this in `Theme.tsx` so it applies to all of our dialogs?

Reply:
Done